app/coffee/SessionMiddleware.coffee

```diff
@@ -2,9 +2,12 @@ Settings = require("settings-sharelatex")
 session = require("express-session")
 MongoStore = require('connect-mongo')(session)
 
+SESSION_TTL_SECS = 6 * 60 * 60   # 6 hours
```

Comment:
I think we could probably go lower on this --- people won't be using this site for very long, hopefully, to grab their projects. One hour?